src/common/components/PlayForms.jsx

```diff
@@ -1,91 +1,123 @@
-import {TextField , FormControl, Autocomplete} from '@mui/material';
-import { useEffect, useState } from 'react';
-import * as _ from 'lodash';
-const PlayForm = ({fields, data, onChange}) => {
+import { useEffect, useState } from "react";
+import { TextField, FormControl, Autocomplete, Button } from "@mui/material";
+import * as _ from "lodash";
 
-  const [formData, setFormData] = useState({})
-  useEffect(() => {
-    setFormData({...data})
-  }, [data])
+const disabledFields = ['issue', 'github']
 
-  const onDataChanged = (key, value) => {
-    formData[key] = value;
-    setFormData({...formData})
-    if(onChange) {
-      onChange(formData)
-    }
-  }
+const PlayForm = ({ fields, formDataObj, onSubmit, isEditPlay}) => {
 
-   const renderField = field => {
-    switch(field.type){
-      case 'input': return <TextField id={field.id} 
-                                      label={field.plaeholder}
-                                      value={formData[field.datafields]} 
-                                      size="small"
-                                      className='w-full'
-                                      {...field}
+  const [formData, setFormData] = useState({...formDataObj});
```

Comment:
```suggestion
  const [formData, setFormData] = useState(formDataObj);
```

Reply:
formDataObj is the default State. it cannot be our main state.